src/Controllers/GagePageController.ts

```diff
@@ -966,144 +1119,288 @@ module StreamStats.Controllers {
+            type: null,
+            title: {
+                text: 'River Discharge (cfs)'
+            },
+            custom: {
+                allowNegativeLog: true
+            }
+        },
+        yAxis: {
+            title: {
+                text: 'River Stage (ft)'
+            },
+            custom: {
+                allowNegativeLog: true
+            },
+            plotLines: [{value: null, color: null, width: null, zIndex: null, label: {text: null}, id: 'plotlines'}]
+        },
+        series  : [
+        {
+            name    : 'USGS Rating Curve',
+            showInNavigator: false,
+            tooltip: { headerFormat:'<b>USGS Rating Curve</b>',
+                pointFormatter: function(){
+                    if (this.dischargeObj !== null){
+                        let discharge = this.x;
+                        let stage = this.y;
+                        return '<br>Gage Height: <b>' + stage + ' ft' + '</b><br>Discharge: <b>' + discharge + ' cfs'
                     }
                 }
-                if (dailyValuesOnly.some(v => v > 0)) {
-                    return {
-                        type: 'logarithmic',
-                        min: null, 
-                        max: null,
-                        stops: [                     
-                        [0 ,   '#FF0000'],
-                        [0.3, '#FFCC33'],
-                        [0.8, '#66CCFF'],
-                        [1 ,   '#3300CC']
-                        ],
-                        startOnTick: false,
-                        endOnTick: false,
-                        labels: {
-                            format: '{value} ft³/s'
-                        },
-                        allowNegativeLog: true
+            },
+            turboThreshold: 0, 
+            type    : 'spline',
+            color   : 'black ',
+            data    : this.dischargeObj,
+            marker: {
+                symbol: 'square',
+                radius: 2.5
+            },
+            showInLegend: this.dischargeObj.length > 5
+        },
+        {
+            name    : 'Annual Peaks',
+            showInNavigator: false,
+            tooltip: {
+                headerFormat:'<b>Annual Peaks</b>',
+                pointFormatter: function(){
+                    if (this.formattedPeakDates !== null){
+                        let UTCday = this.getUTCDate;
+                        let year = this.getUTCFullYear;
+                        let month = this.getUTCMonth;
+                            month += 1; // adding a month to the UTC months (which are zero-indexed)
+                        let formattedUTCPeakDate = month + '/' + UTCday + '/' + year;
+                        let discharge = this.x;
+                        let stage = this.y;
+                        let peakDate = this.date
+                        return '<br>Date <b>' + peakDate +  '<br>Peak <b>' + discharge + ' cfs' + '<br>at stage <b>' + stage + ' ft'
```

Comment:
Making some stylistic changes to be more consistent with pop-ups in other plots
```suggestion
                        return '<br>Date: <b>' + peakDate +  '</b><br>Peak: <b>' + discharge + ' cfs</b><br>at stage <b>' + stage + ' ft</br>'
```